### PR TITLE
Magic action "/request-release"

### PR DESCRIPTION
**Pull request by OskarStark**
Use `/request-release` for:
CleanShot 2020-11-04 at 15 48 42@2x

### CODE REVIEW

File: templates/github/request-release.markdown [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Ok @#handle#,
+
+I requested a new release for **#project#** via Slack in `#releases` channel :+1:
+
+_If you want to get notified about new releases, make sure to follow [SonataNews](https://twitter.com/sonatanews) on Twitter!_
```
Comment (core23) [inline templates/github/request-release.markdown:5]: Maybe we can integrate this via the Twitter API. The account uses IFTTT at the moment.

File: src/Github/HookProcessor.php [hunk 2]
```diff
@@ -85,6 +89,105 @@ public function processReviewLabel(Payload $payload): void
         );
     }
 
+    /**
+     * Performs an action and reports back via comment:.
+     *
+     * 1) "/request-release" - requests a release of a bundle in Slack 'releases channel
```
Comment (greg0ire) [inline src/Github/HookProcessor.php:95]:
```suggestion
     * 1) "/request-release" - requests a release of a bundle in Slack's #releases channel
```

Comment (OskarStark) [reply src/Github/HookProcessor.php]: Thanks

**Pull request merged**